app/components/ui/Navigation/DesktopMenuItems.tsx

```diff
@@ -103,7 +45,7 @@ const DesktopMenuItems = ({
       key={uniqueKey}
       to={menuItem.url}
       className={styles.navigationMenuLink}
-      onClick={() => setActiveDesktopSubMenu(null)}
+      onClick={() => handleMenuToggle(null)}
```

Comment:
Does this toggle actually do anything anymore since you've added the other behaviors to handle clicks outside the menu?